String-is.md

```diff
@@ -0,0 +1,20 @@
+# String.is
+## motivation
+проверяет переданный параметр на то, что он является строкой.
+
+по аналогии с методом Array.is
```

Comment:
checks if the passed parameter  is a string.

by analogy with the Array.is method